tech/languages/python/multiple-pythons.md

```diff
@@ -48,6 +48,11 @@ packages only. Other CPython versions might be **unstable** or even **dangerous*
 (either because they are extremely old or quite the contrary alpha/beta quality)
 and are intended solely for development.
 
+**\*** Interpreters marked with \* do not work with Fedora packaged Tox and
+virtualenv, because they are extremely old.
```

Comment:
hmm. packages or interpreters are old? how to say ti better?

Reply:
How about you move it into a separate list?

`Interpreters known not to work with Tox:`

or similar.